lib/frame_processor.py

```diff
@@ -210,6 +210,14 @@ def process_chunk(self):
             # Check if there are new parameters
             new_parameters = self.parameters_queue.get_nowait()
             if new_parameters != self.parameters:
+                # Clear stale transition when new prompts arrive without transition
```

Comment:
I suspect we can simplify things here and wrote up some thoughts in https://github.com/daydreamlive/scope/issues/134.

It represents a bigger change though so rather than make smaller changes here to try to have cleaner logic my thinking is we can do a follow up for that issue outside of this PR.